### PR TITLE
added ui-router

### PR DESCRIPTION
**Pull request by jaguisanda09**
@benjbuluran @gybers @jezmaghuyop @albertdesalisa @Azaxan  @felix-csm @jhenneson @pnxlee @perezavidien @amielmagenic @rolandj111 Kindly check my implementation for routing using ui.router

### CODE REVIEW

File: Magenic.Manpower/Magenic.Manpower.csproj [hunk 1]
```diff
@@ -66,10 +66,16 @@
     <Reference Include="System.EnterpriseServices" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="js\controllers\LoginController.js" />
     <Content Include="packages.config" />
+    <Content Include="Scripts\angular-route.js" />
+    <Content Include="Scripts\angular-route.min.js" />
```
Comment (aaa-magenic-amielm) [inline Magenic.Manpower/Magenic.Manpower.csproj]: maybe we can remove the angular-route if we're using the ui-route?

**Review by felix-csm — COMMENTED** [review]
Can we segragate files per module, like, js/login/..., then put all login related files?

File: Magenic.Manpower/Magenic.Manpower.csproj (hunk 1 above)
Comment (jaguisanda09) [reply Magenic.Manpower/Magenic.Manpower.csproj]: @amielmagenic yes :) I will remove that

**Comment by jaguisanda09** [comment]
@felix-csm are you referring to modular design sir? I think, yes, so we will be having module folder, then inside this is each folder for each module

**Review by jhenneson — APPROVED** [review]

**Review by aaa-magenic-amielm — APPROVED** [review]

**Comment by jaguisanda09** [comment]
merging to master, so the rest can use the routing

**Pull request merged**